packages/ramldoc-generator/src/main.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs');
+const path = require('path');
+const shell = require('shelljs');
+const chalk = require('chalk');
+
+function generateRaml(options) {
+  const resolvedSourcePath = path.resolve(options.apiSpecSourcePath);
+
+  // Check if the spec file exists.
+  if (!fs.existsSync(resolvedSourcePath)) {
+    console.error(
+      `"${resolvedSourcePath}" does not exist`,
+      chalk.red.bold('ERROR')
+    );
+    process.exit(1);
```

Comment:
NOTE: usually the library implementation should throw an `Error` and the file handling the CLI part should catch it and exit the process.